app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/AutoCloseSearchRequestJob.cs

```diff
@@ -57,7 +57,15 @@ public async Task Execute(IJobExecutionContext context)
                         ssgSearchRequest.AutoCloseStatus == SearchRequestAutoCloseStatusCode.CPMissingData.Value)
                     {
                         _logger.LogInformation("Autoclose: Calling ssg_SearchRequestCreateCouldNotAutoCloseNote SearchRequestId=" + ssgSearchRequest.SearchRequestId + " AutoCloseStatus "+ssgSearchRequest.AutoCloseStatus+"");
-                        await _searchRequestService.SearchRequestCreateCouldNotAutoCloseNote(ssgSearchRequest.SearchRequestId);
+                        try
+                        {
+                            await _searchRequestService.SearchRequestCreateCouldNotAutoCloseNote(ssgSearchRequest.SearchRequestId, cts.Token);
+                        }
+                        catch (Exception e)
```

Comment:
Are there know reasons?  Having a comment indicating what causes have been done and what the side effect of just skipping this record.  If the problem is not transient, could the number of search requests needing to be auto closed continue to increase until there are too many?